packages/material-ui-styled-engine-sc/src/GlobalStyles/GlobalStyles.test.js

```diff
@@ -101,4 +101,12 @@ describe('GlobalStyles', () => {
       color: 'rgb(255, 0, 0)',
     });
   });
+
+  it('should raise console.error()', function test() {
+    // if (/jsdom/.test(window.navigator.userAgent)) this.skip();
+
+    expect(() => {
+      styled('span')({ color: undefined });
```

Comment:
```suggestion
      styled('span')();
```

Aren't we suppose to fix this scenario (no arguments)?